src/argilla/client/feedback/dataset/local/dataset.py

```diff
@@ -15,15 +15,15 @@
 from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Optional, Union
 
 from argilla.client.feedback.constants import FETCHING_BATCH_SIZE
-from argilla.client.feedback.dataset.base import FeedbackDatasetBase
-from argilla.client.feedback.dataset.mixins import ArgillaMixin
+from argilla.client.feedback.dataset.local.base import FeedbackDatasetBase
+from argilla.client.feedback.dataset.local.mixins import ArgillaMixin
 from argilla.client.feedback.schemas.types import AllowedFieldTypes, AllowedQuestionTypes
 
 if TYPE_CHECKING:
     from argilla.client.feedback.schemas import FeedbackRecord
 
 
-class FeedbackDataset(FeedbackDatasetBase, ArgillaMixin):
+class FeedbackDataset(ArgillaMixin, FeedbackDatasetBase):
```

Comment:
Can we revert the order here? I liked most the class/es first, then the mixins

```suggestion
class FeedbackDataset(FeedbackDatasetBase, ArgillaMixin):
```